mgmtsystem_nonconformity/__manifest__.py

```diff
@@ -33,23 +16,24 @@
         'security/ir.model.access.csv',
         'security/mgmtsystem_nonconformity_security.xml',
         'views/mgmtsystem_nonconformity.xml',
-        'views/mgmtsystem_nonconformity_stage.xml',
         'views/mgmtsystem_origin.xml',
         'views/mgmtsystem_cause.xml',
         'views/mgmtsystem_severity.xml',
         'views/mgmtsystem_action.xml',
-        'reports/mgmtsystem_nonconformity_report.xml',
+        'views/mgmtsystem_nonconformity_stage.xml',
         'data/sequence.xml',
         'data/mgmtsystem_nonconformity_severity.xml',
         'data/mgmtsystem_nonconformity_origin.xml',
         'data/mgmtsystem_nonconformity_cause.xml',
         'data/mgmtsystem_nonconformity_stage.xml',
         'data/mail_message_subtype.xml',
+        'reports/mgmtsystem_nonconformity_report.xml',
     ],
+    "images": [],
```

Comment:
@dreispt  youre right, i did an additional commit, this tag slipped inside becouse i still used the __openerp__py from v. 8.0.

Anyways should i rebase since theres one aditional commit ?